src/manifest/manifest_test.ts

```diff
@@ -1433,3 +1433,120 @@ Deno.test("SlackManifest() oauth2 providers get set properly with token_url_conf
     true,
   );
 });
+
+Deno.test("SlackManifest() oauth2 providers get set properly with identity_config", () => {
```

Comment:
NIT: This file seems large and there may be more tests added to it in the future
Maybe we could create `src/providers/oauth2/mod_test.ts` that would contain tests related to the `src/providers/oauth2/` folder 💡 

Let me know what you think, I am unsure if this will trigger a larger refactor of tests